### PR TITLE
Bug 1959564: move verify run test from tools to cli image to avoid additional /run content

### PR DESCRIPTION
**Pull request by gabemontero**
@adambkaplan @nalind FYI

Assuming we can get this to work like we want it I'll associate the bz, fully remove the skip, etc.

### CODE REVIEW

File: test/extended/builds/run_fs_verification.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ spec:
           value: "10"
       imageOptimizationPolicy: SkipLayers
     type: Docker
-`, image.ShellImage())
+`, image.LimitedShellImage())
```
Comment (gabemontero) [inline test/extended/builds/run_fs_verification.go:38]: Or as @nalind mentioned during scrum, the cli image is built off of 

`registry.ci.openshift.org/ocp/4.8:base`

There is not helper method yet in the image component that exposes that image, but we could either try to add one there, or just reference it directly here.

Comment (gabemontero) [reply test/extended/builds/run_fs_verification.go:38]: But it gets complicated cause we really should use an image stream image reference, and there is not imagestream for the base image right now.

This gets into the barriers we may encounter that I alluded to wrt adding imagestreams to the openshift namespace.
Similarly getting `registry.ci.openshift.org/ocp/4.8:base` into https://github.com/openshift/origin/blob/master/test/extended/util/image/image.go#L15 could prove difficult.

IMO the cli image via `LimitedShellImage` is the path of least resistance

**Comment by gabemontero** [comment]
missed a spot where I had to switch

**Comment by gabemontero** [comment]
ok my test cluster is now up, and this is passing for me locally

let's see what happens in CI

**Comment by gabemontero** [comment]
e2e-gcp-builds passes @adambkaplan @nalind

**Comment by gabemontero** [comment]
GCP IAM pain on e2e-gcp failure

/test e2e-gcp

**Comment by gabemontero** [comment]
e2e-gcp still has IAM pain, cluster can't get started

**Comment by gabemontero** [comment]
unrelated failures in e2e-aws-csi and e2e-aws-disruptive

**Comment by gabemontero** [comment]
all green tests for this bz PR @adambkaplan

**Comment by xiuwang** [comment]
The two tests are passed , the result is in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/26164/pull-ci-openshift-origin-master-e2e-aws-fips/1394396549101916160

/label qe-approved

**Review by adambkaplan — APPROVED** [review]
/lgtm

**Comment by gabemontero** [comment]
/cherrypick release-4.7

@dmage per exchange in slack

**Pull request merged**